poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
It seems this was generated with an outdated version of Poetry. I don't think there's any issue in this case, but sometimes there is. I think we probably don't have a clear guideline for adding/updating dependencies but for when we have we should keep in mind that Poetry should be updated before making any changes.